docker-compose-dev.yaml

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
what's the benefit of using docker here as opposed to just running the client locally?

Reply:
The goal is to keep track of the L1 client logs (because they are useful sometimes) without them interfering with the L2 client logs.

Reply:
should we move this to `dev` folder then?

Reply:
Yes we can do that

Reply:
Done in [ee7fc52](https://github.com/lambdaclass/lambda_ethereum_rust/pull/826/commits/ee7fc52620befe7cb1bd8148af31825706779994)